packages/fxa-auth-server/test/scripts/email-config.js

```diff
@@ -265,9 +265,8 @@ describe('scripts/email-config:', () => {
         .execAsync('node -r esbuild-register scripts/email-config read', {
           cwd,
         })
-        .then(({ stdout, stderr }) => {
```

Comment:
16 emits some deprecation warnings to stderr. it's not important to assert stderr